### PR TITLE
Added Merge Sort in PHP

### LINKED ISSUE #2136: Merge Sort
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [x] Dart
- [x] Go
- [x] Java
- [x] JS
- [ ] Kotlin - @shubhammantri1
- [x] Php
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @m-code12
- [x] TypeScript - @m-code12

### PR DESCRIPTION
**Pull request by chandrikadeb7**
Fixes #2136 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Merge Sort implementation in PHP
- 

#### Languages Used:
- PHP
- 

#### Files Added:
- Merge_Sort/Merge_Sort.php
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Merge_Sort/Merge_Sort.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+author: chandrikadeb7
+date: 06.03.2020
+*/
+<?php
+function merge_sort($array)
+{
+	if(count($array) == 1 ) 
+		return $array;
+	$mid = count($array) / 2;
+    $left = array_slice($array, 0, $mid);
```
Comment (jainaman224) [inline Merge_Sort/Merge_Sort.php]: Incorrect indentation.

File: Merge_Sort/Merge_Sort.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/* 
+author: chandrikadeb7
+date: 06.03.2020
+*/
+<?php
+function merge_sort($array)
+{
+	if(count($array) == 1 )
```
Comment (jainaman224) [inline Merge_Sort/Merge_Sort.php]: Use 4 space indentation.

File: Merge_Sort/Merge_Sort.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
+/* 
+author: chandrikadeb7
```
Comment (jainaman224) [inline Merge_Sort/Merge_Sort.php]: Please remove this.

File: Merge_Sort/Merge_Sort.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+	if(count($array) == 1 ) 
+		return $array;
+	$mid = count($array) / 2;
+    $left = array_slice($array, 0, $mid);
+    $right = array_slice($array, $mid);
+	$left = merge_sort($left);
+	$right = merge_sort($right);
+	return merge($left, $right);
+}	
+function merge($left, $right)
```
Comment (jainaman224) [inline Merge_Sort/Merge_Sort.php]: Add a newline above function definition.

File: Merge_Sort/Merge_Sort.php (hunk 2 above)
Comment (jainaman224) [reply Merge_Sort/Merge_Sort.php]: Not done.

**Comment by chandrikadeb7** [comment]
@somya-kapoor @jainaman224  Please review.

**Comment by chandrikadeb7** [comment]
@raghav-dalmia please review

**Pull request closed without merging**